client/ayon_core/pipeline/publish/abstract_collect_render.py

```diff
@@ -214,8 +217,11 @@ def process(self, context):
             data = self.add_additional_data(data)
             render_instance_dict = attr.asdict(render_instance)
 
-            instance = context.create_instance(render_instance.name)
-            instance.data["label"] = render_instance.label
```

Comment:
Just so it's clear - I removed this particular line because `label` would also be in `render_instance_dict` and already gets merged into the instance later anyway.